pkg/adsc/adsc.go

```diff
@@ -94,6 +96,10 @@ type Config struct {
 	// XDSSAN is the expected SAN of the XDS server. If not set, the ProxyConfig.DiscoveryAddress is used.
 	XDSSAN string
 
+	// XDSRootCAFile explicitly set the root CA to be used for the XDS connection.
+	// Mirrors Envoy file.
+	XDSRootCAFile string
```

Comment:
If this new option is merged, should it be exposed in _istioctl_?

Reply:
The 'CA' and XDS server may use different certs. For example CA may be an external integration.

In particular - if the XDS server uses ACME DNS certs, this will be the platform cert (/etc/certs...)